docs/communication_protocols/dronecan_protocol.rst

```diff
@@ -105,11 +105,20 @@ protocol being tunneled, a channel ID allowing for additional routing options, a
 Vertiq modules are configured to accept broadcast messages with a protocol value of 0x55, and check the received data buffer for valid :ref:`IQUART data <uart_messaging>`. 
 As such, the DroneCAN tunnel broadcast message allows users to transmit IQUART messages via DroneCAN. Vertiq modules do not support use of the channel ID for additional routing.
 
-A very rudimentary example of using the DroneCAN tunnel message is provided below. The example requires an SLCAN device such as the `Zubax Babel <https://zubax.com/products/adapters/canface>`_, and two :ref:`FTDI devices <connection_guide>`. 
-The hardware is configured as follows:
+A very rudimentary example of using the DroneCAN tunnel message is provided below. The example requires an SLCAN device such as the `Zubax Babel <https://zubax.com/products/adapters/canface>`_ and any 
```

Comment:
I'm not asking you to go do a Linux example, but you should specify here that this is just a Windows example. COM ports are a Windows only thing, well at least they aren't called that on Linux, and com0com is just for Windows I think. Just want to be clear that this is just a Windows walkthrough, and if you're on Linux you can use this as a basis to go figure it out yourself.